src/plugins/CMakeLists.txt

```diff
@@ -2,30 +2,12 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-if(ENABLE_HETERO)
-    add_subdirectory(hetero)
-endif()
-
-if(ENABLE_INTEL_CPU)
-    add_subdirectory(intel_cpu)
-endif()
-
-if(ENABLE_INTEL_GPU)
-    add_subdirectory(intel_gpu)
-endif()
-
-if(ENABLE_INTEL_GNA)
-    add_subdirectory(intel_gna)
-endif()
-
-if(ENABLE_INTEL_MYRIAD_COMMON)
-    add_subdirectory(intel_myriad)
-endif()
-
-if(ENABLE_AUTO OR ENABLE_MULTI)
-    add_subdirectory(auto)
-endif()
-
-if(ENABLE_AUTO_BATCH)
-    add_subdirectory(auto_batch)
-endif()
+file(GLOB local_modules "${CMAKE_CURRENT_SOURCE_DIR}/*")
+foreach(module_path IN LISTS local_modules)
+    if(module_path)
```

Comment:
why do we need to check it? and why module_path may not contain cmakelists.txt file?

Reply:
Ok, will remove 😄